h2o-py/h2o/frame.py

```diff
@@ -1221,35 +1221,44 @@ def structure(self):
                 print("num {}".format(" ".join(it[0] if it else "nan" for it in h2o.as_list(self[:10, i], False)[1:])))
 
 
-    def as_data_frame(self, use_pandas=True, header=True):
+    def as_data_frame(self, use_pandas=True, header=True, na_value=''):
```

Comment:
Yes, the previous behavior is incorrect. It is not what a user would expect.

We don't need to be backwards compatible in this case.